solutions/cf-1528A.mdx

```diff
@@ -0,0 +1,110 @@
+---
+id: cf-1528A
+source: CF
+title: Parsa's Humongous Tree
+author: Ryan Chou
+---
+
+[Official Editorial (C++)](https://codeforces.com/blog/entry/91058)
+
+## Explanation
+
+Let's go through a case where we've already computed the beauty values for all of vertex $u$'s neighbors.
```

Comment:
emphasize that this is a 100% hypothetical scenario
"say, hypothetically, for the sake of the argument, that we already computed all the best beauty values for all of vertext..."